src/storage/sql/migrations/versions/1bf4e4e7b24f_initial_schema.py

```diff
@@ -0,0 +1,103 @@
+"""Initial schema
+
+Revision ID: 1bf4e4e7b24f
+Revises:
+Create Date: 2020-07-26 08:46:28.752972
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '1bf4e4e7b24f'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'snippets',
+
+        # an internal autoincrementing identifier used in foreign keys.
+        # Normally not visible publicly, except for the case when it is
+        # used for looking legacy snippets up by id
```

Comment:
I was actually thinking that for legacy snippets we can set `slug` to be their old `id` and avoid having special logic in the code base. What do you think?

Reply:
Oh, that's an interesting point. We could treat them as strings, indeed! I want to keep the integer IDs, though, as they are a better fit for foreign key constraints (because they are smaller and, thus, the indexes are smaller, and also comparing 4-byte integers is likely slightly more efficient).

Reply:
Integer IDs are OK. :+1: Another question I have is what are we going to do with both `slug` and `id` in term of API response? Should we use `slug` as an `id` in API response? Or you want to explicitly separate these two for API users?

Reply:
My plan was to treat integer IDs as an implementation detail and not expose them via the API. Perhaps, only put `slug` to the response under the name `id`?